app/Controllers/ReservationManager.php

```
@@ -66,6 +69,23 @@ public function flushDeferredNotifications(): void
                 ]);
             }
         }
+
+        // Stesso pattern per le notifiche di scadenza coda accodate da
+        // cancelExpiredReservations() in transazione esterna (M11).
+        $pendingExpiry = $this->deferredExpiryNotifications;
+        $this->deferredExpiryNotifications = [];
+        if ($pendingExpiry !== []) {
+            $notificationService = new \App\Support\NotificationService($this->db);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '78', column '40'). (undefined)

(MissingImport)